server/controllers/users/validate.js

```diff
@@ -0,0 +1,11 @@
+import Joi from "@hapi/joi";
+import joiError from "../../helpers/joiError";
+
+export default (req, res, next) => {
+  const { email, password } = req.body;
+  const schema = Joi.object().keys({
+    email: Joi.string().required(),
+    password: Joi.string().required()
```

Comment:
Why not force the user to use alphanumeric passwords for more security than a  string. so just implement an alphanumeric validation using joi

Reply:
because it's on the login route, users don't need to know the pattern for the password to eliminate awareness, I will for sure do that(Joi regex for alphanums) on the signup route but I don't think that it's a good idea on the login route